public/locales/en-US/translations.json

```diff
@@ -29,6 +29,7 @@
   "transaction_short": "TX",
   "nodes": "Nodes",
   "validators": "Validators",
+  "upgrade-status": "Upgrade Status",
```

Comment:
Should have an underscore.  `Tabs.tsx` can be updated with `{t(title.replace('-', '_'))}` like in #270.

Reply:
I will just replace - with _ for now. Tabs can be updated once #270 get merged